ml-proto/host/script.ml

```diff
@@ -21,35 +21,20 @@ type script = command list
 
 (* Execution *)
 
+module Syntax = Error.Make ()
+module AssertFailure = Error.Make ()
+
+exception Syntax = Syntax.Error
+exception AssertFailure = AssertFailure.Error
```

Comment:
Can you add a comment explaining that this AssertFailure is for failures in asserts in the .wast script, rather than assertions in the spec code itself?